dlx_rest/static/js/components/search.js

```diff
@@ -340,9 +340,12 @@ export let searchcomponent = {
             <div v-if="records.length > 0 && records.length === totalCount">End</div>
             <div v-else-if="records.length > 0">
                 <span>{{records.length}} / {{totalCount}} loaded</span>                 
-                <i class="spinner-border mr-2">
+                <i class="spinner-border mr-2"></i>
             </div>
+            </div>&nbsp;</div> <!-- padding -->
```

Comment:
Just noting we can do this with margin (m) and padding (p) classes in bootstrap. For example, increasing the padding around all dimensions of an element by one unit: `class="p-1"`; just vertical: `class="py-1"`; just the bottom: `class="pb-1"`; etc.